Sources/SQLKit/Query/SQLQueryString.swift

```diff
@@ -119,6 +119,10 @@ extension SQLQueryString {
     public static func +(lhs: SQLQueryString, rhs: SQLQueryString) -> SQLQueryString {
         return "\(lhs)\(rhs)"
     }
+    
+    public static func +=(lhs: inout SQLQueryString, rhs: SQLQueryString) {
+        lhs = "\(lhs)\(rhs)"
```

Comment:
```suggestion
        lhs.fragments.insert(rhs,.fragments, at: lhs.fragments.startIndex)
```
This avoids the creation of an unnecessary intermediate object.

Reply:
Thanks!